tests/requests.test.ts

```diff
@@ -107,3 +107,41 @@ test('intercepts a POST request with array buffer body', async ({
   expect(request.url).toBe('http://localhost:5173/action')
   await expect(request.text()).resolves.toBe('hello world')
 })
+
+test('intercepts an asset request when interceptCommonAssetRequests is false', async ({ network, page }) => {
+    network.use(
+      http.get('/index.html', () => {
+        return HttpResponse.text('NOT VALID HTML')
+      }),
+    )
+
+    await page.goto('/')
+    const htmlContent = await page.evaluate(async () => {
+      const res = await fetch('/index.html')
+      return res.text()
+    })
+
+    expect(htmlContent).toContain('NOT VALID HTML')
+  })
+
+  const testIgnoreAssets = testBase.extend<Fixtures>({
+    network: createNetworkFixture({ ignoreCommonAssetRequests: true }),
+  })
+
+  testIgnoreAssets('passes through an asset request when interceptCommonAssetRequests is true', async ({ network, page }) => {
```

Comment:
One downside of enabling this behavior is that asset requests can no longer be mocked at all. Is that acceptable? Maybe there's a way we can modify the API of `network.use` to allow overriding default fixture options as well as passing handlers?